heartbeat/monitors/active/http/simple_transp.go

```diff
@@ -191,7 +178,7 @@ func (t *SimpleTransport) readResponse(
 
 	t.sigStartRead()
 
-	if requestedGzip && resp.Header.Get("Content-Encoding") == gzipEncoding {
+	if resp.Header.Get("Content-Encoding") == gzipEncoding {
```

Comment:
I think this behavior should make the new test do the right thing, which is to always decode if necessary, regardless of whether the `Accept` header was sent.

Reply:
Will implement after we clarify reqs in the previous comment.